app/controllers/locations_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
     @location = Location.new(location_params)
 
     if @location.save
-      @location.geocoding.operate_later
+      @location.geocoding.perform_later
```

Comment:
I do think `operate` was a nice term from the perspective of `ActiveOperator`, though `perform_later` seems like it feels more obvious to the user.

Reply:
Yeah, I was on the fence, but I felt like after collapsing `Operator` and `Operation`, it made more sense to say "perform operation"...could be easily persuaded on this though.